lib/pbench/test/unit/agent/test_tool_meister_stop.py

```diff
@@ -0,0 +1,40 @@
+"""Tests for the Tool Meister "stop" module.
+"""
+from pbench.agent.tool_meister_stop import RedisServer
+
+
+class TestRedisServer:
+    """Verify the RedisServer class use by Tool Meister "stop"."""
+
+    def test_locally_managed(self, tmp_path):
+        # Locally managed means we have a run directory, ...
+        rundir = tmp_path / "run-dir"
+        rundir.mkdir()
+        # ... with a "tm" sub-directory, ...
+        tmdir = rundir / "tm"
+        tmdir.mkdir()
+        # ... containing a "redis.pid" file.
+        pidfile = tmdir / "redis.pid"
+        pidfile.write_text("12345")
+
+        rs = RedisServer("", rundir, "notme.example.com")
+        assert (
+            rs.locally_managed()
+        ), "Populated run directory did not result in a 'locally managed' RedisServer instance"
```

Comment:
"Populated"? Since we actually care about `redis.pid` this seems unnecessarily indirect.

Reply:
We do care about `redis.pid`, but that must be in a sub-directory, `tm`, and so with the two things that were created I used populated.